task1/src/csv.js

```diff
@@ -0,0 +1,48 @@
+import { appendFileSync, existsSync, unlinkSync } from 'fs';
+import { csv } from 'csvtojson';
+
+const csvFilePath = './csv/input.csv';
+const txtFilePath = './txt/output.txt';
+
+deleteTxtFile(txtFilePath);
+convertCsvData();
+
+function convertCsvData() {
+  csv()
+    .fromFile(csvFilePath)
```

Comment:
Here I can see a third point of evaluation criteria - file is fully loaded into RAM and then its written line by line.
But fourth one isn't here - reading from file line by line(or chunk by chunk) and pipeline is not used.

Reply:
added method to read csv file by stream